feathr_project/feathr/materialization_settings.py

```diff
@@ -0,0 +1,32 @@
+from abc import abstractclassmethod
+from datetime import datetime, timedelta
+from typing import List
+from feathr.sink import Sink
+
+
+class BackfillTime:
+    """Time range to materialize/backfill feature data.
+    Attributes:
+        start: start time of the backfill, inclusive.
+        end: end time of the backfill, inclusive.
+        step: duration of each backfill step. e.g. if daily, use timedelta(days=1)
```

Comment:
i don't see step being used.

Reply:
it's a bit hard to understand how to use step. Please provide more explanations.

Reply:
It is used in the feature materialization api